--- backfire_10.03.1.orig/feeds/packages/net/tor/Makefile	2012-03-15 17:30:51.828807949 +0000
+++ backfire_10.03.1/feeds/packages/net/tor/Makefile	2012-03-15 16:52:51.045618251 +0000
@@ -8,18 +8,33 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tor
-PKG_VERSION:=0.2.1.30
+PKG_VERSION:=0.2.2.35
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.torproject.org/dist \
 	https://archive.torproject.org/tor-package-archive
-PKG_MD5SUM:=6c6d61e053af5969a245d025c4cfce9d
+PKG_MD5SUM:=dcecf699c4b929319d5f1ce0358d4835
 
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
+define Package/tor/description
+$(call Package/tor/Default/description)
+ This package contains the tor daemon.
+endef
+
+define Package/tor-geoip
+$(call Package/tor/Default)
+  DEPENDS:=+tor
+endef
+
+define Package/tor-geoip/description
+$(call Package/tor/Default/description)
+ This package contains a GeoIP database mapping IP addresses to countries.
+endef
+
 define Package/tor/Default
   SECTION:=net
   CATEGORY:=Network
